fdbserver/workloads/BlobGranuleVerifier.actor.cpp

```diff
@@ -80,8 +80,8 @@ struct BlobGranuleVerifierWorkload : TestWorkload {
 		sharedRandomNumber /= 2;
 		// randomly some tests write data first and then turn on blob granules later, to test conversion of existing DB
 		// TODO CHANGE to infrequent!
```

Comment:
Should you delete this comment after adding `&& sharedRandomNumber % 10 == 0;`? Also might as well delete the line you commented out?